File: 344. Reverse String.py
class Solution(object):
	def reverseString(self,s):
		return s[::-1]


    def reverseString(self, s):
        l = len(s)
        if l < 2:
            return s
        return self.reverseString(s[l/2:]) + self.reverseString(s[:l/2])

File: 344. Reverse String.java
class Solution{
	public String reverString(String s){
		StringBuilder ret = new StringBuilder(s);
		return ret.reverse().toString();
	}
}

	public String[] findWords(String s){
		char[] word = s.toCharArray()
		int i=0;
		int j = s.length();
		while (i < j){
			char temp = word[i];
            word[i] = word[j];
            word[j] = temp;
            i++;
            j--;
		}
		return new String(word)
	}



	public String reverseString(String s) {
        byte[] bytes = s.getBytes();
        int i = 0;
        int j = s.length() - 1;
        while (i < j) {
            byte temp = bytes[i];
            bytes[i] = bytes[j];
            bytes[j] = temp;
            i++;
            j--;
        }
        return new String(bytes);
    }